### PR TITLE
add analytics script to Build and Run

### PR DESCRIPTION
**Pull request by mariakleiner**
(and a little polymath related fix sneaked in))

### CODE REVIEW

File: pkg/Library/Polymath/Atoms/AskPolymath.js [hunk 1]
```diff
@@ -13,6 +13,6 @@ async update({library, query}, state, {service, isDirty}) {
     return;
   }
   const result = await service('PolymathService', 'Ask', {library, query});
-  return {query, result, completion: result.completion};
+  return {query, result, completion: result?.completion ?? 'n/a'};
```
Comment (sjmiles) [inline pkg/Library/Polymath/Atoms/AskPolymath.js:16]: Nit: as a rule, factor it out if there is logic.

File: pkg/Build/index.html [hunk 2]
```diff
@@ -3,6 +3,18 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no, viewport-fit=cover">
 <meta name="apple-mobile-web-app-capable" content="yes">
+
+<head>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KXNNFC60KV"></script>
```
Comment (sjmiles) [inline pkg/Build/index.html:8]: I find this a little off-putting wrt our user-sovereignty story. 

I'm not sure how to square this circle, but we should discuss.

**Review by sjmiles — APPROVED** [review]

File: pkg/Library/Polymath/Atoms/AskPolymath.js (hunk 1 above)
Comment (mariakleiner) [reply pkg/Library/Polymath/Atoms/AskPolymath.js]: factored

File: pkg/Build/index.html (hunk 2 above)
Comment (mariakleiner) [reply pkg/Build/index.html:8]: discussed :)

**Pull request merged**